lib/matplotlib/_mathtext.py

```diff
@@ -538,8 +506,7 @@ def _get_glyph(self, fontname, font_class, sym, fontsize, math=True):
                 glyphindex = font.get_char_index(uniindex)
```

Comment:
Admittedly helped from LGTM, but since `glyphindex` is no longer used, neither that or `uniindex` are required here.

Reply:
Good catch, edited accordingly.